### DISCUSSION TITLE
Adding a Terrain editor

### DISCUSSION THREAD

**Discussion by yeet227** [godotengine/godot-proposals#6112] [3D]
As the title says could adding a terrain generator and editor with brushes or at least adding a noise interpreter for terrain generator to be imported

**Comment by HeadClot** [comment]
I want to chime in on this. This is very badly needed in Godot 4.x

I know that there is a Godot Native module for godot 3.x that supports [height mapped terrain](https://github.com/ozzr/godot_terrain) however it does not support Godot 4.x

**Comment by Zireael07** [comment]
Duplicate of #445
